src/resources/i18n/translations/en.json

```
@@ -419,42 +418,42 @@
 	"No summary available": "No summary available.",
 	"No thanks": "No thanks",
 	"No": "No",
-	"Now Playing Info": "\"Now playing\" info",
-	"Now Playing": "Now playing",
+	"Now Playing Info": "\"Now Playing\" Info",
+	"Now Playing": "Now Playing",
 	"OFFLINE MODE ENABLED": "Offline mode on",
 	"OK": "OK",
-	"Official Episode Page": "Official episode page",
-	"Official Podcast Page": "Official podcast page",
-	"Offline Mode": "Offline mode",
+	"Official Episode Page": "Official Episode Page",
+	"Official Podcast Page": "Official Podcast Page",
+	"Offline Mode": "Offline Mode",
 	"Offline mode can save battery life and improve performance": "Can save battery life and improve performance in areas with poor connectivity.",
 	"Ok": "OK",
-	"Only allow downloading episodes when connected to Wifi": "Wi-Fi only",
-	"Only with Link means only people who": "\"Only with link\" means it not in the \"Public\" list on Podverse and the clip's link is needed to play it. Create public clips with Podverse premium.",
+	"Only allow downloading episodes when connected to Wifi": "Only allow downloading episodes when connected to Wifi",
+	"Only with Link means only people who": "\"Only with link\" means it not in the \"Public\" list on Podverse and the clip's link is needed to play it. Create public clips with Podverse Premium.",
 	"Only with link": "Only with link",
 	"Other": "Other",
 	"Password requirements": "Password requirements:",
 	"Password": "Password",
 	"Pause": "Pause",
-	"Permission Denied": "Permission denied",
+	"Permission Denied": "Permission Denied",
 	"Play": "Play",
 	"Played episode": "Played episode",
 	"Player": "Player",
 	"Playlist Not Found": "Could not find the playlist",
 	"Playlist title": "Playlist title",
 	"Playlist": "Playlist",
 	"Playlists": "Playlists",
-	"Please check your inbox If this address exists in our system you should receive a reset password email shortly": "You should get an email to reset your password shortly if it is the right e-mail address.",
-	"Please check your internet connection and try again later": "Ensure you are connected to the Internet and try again.",
+	"Please check your inbox If this address exists in our system you should receive a reset password email shortly": "You should get an email to reset your password shortly if the e-mail is in our database.",
+	"Please check your internet connection and try again later": "Please check your internet connection and try again later.",
```

```suggestion
	"Please check your internet connection and try again later": "Please connect to the internet and try again later.",
```
I posit users will know the Internet is a network and where it comes from in the capacity they are capable of doing something about it.

---

👍